Lib/fontmake/instantiator.py

```diff
@@ -224,21 +226,33 @@ def from_designspace(
         # because the math behind varLib and MutatorMath uses the default font as the
         # point of reference for all data.
         default_font = designspace.default.font
+        non_default_layer_name = designspace.default.layerName
+        non_default_layer_flag = (
```

Comment:
instead of `non_default_layer_flag`, how about `default_source_uses_non_default_layer`.. a bit long maybe but it kind of reads better when you see it in e.g. `if default_source_uses_non_default_layer:`

Reply:
OK, fixed.